spec/features/invoices/index_spec.rb

```diff
@@ -35,4 +35,21 @@
       expect(subject).to_not have_content(org2_invoice.customer)
     end
   end
+
+  context 'colorize invoice' do
+    let!(:overdue_invoice) { create :invoice, organization: org, ends_at: Date.current - 16.days }
+    let!(:paid_invoice) { create :invoice, organization: org, paid_at: Date.current }
+
+    before do
+      visit invoices_path
+    end
+
+    it "overdue invoice has class 'overdue'" do
+      expect(subject).to have_css('tr.invoice.overdue')
```

Comment:
@yunixon need to check that you see class `overdue` for specific item in the list.

Use:
`within "#invoice_list" do
    # check dom element with correct class and id dom_id(overdue_invoice)
end
```

and same for other `:paid_invoice`.